src/server/server_family.cc

```diff
@@ -872,6 +884,30 @@ void PrintPrometheusMetrics(const Metrics& m, StringResponse* resp) {
                       &db_key_expire_metrics);
   }
 
+  // Command stats
```

Comment:
we should first of all to provide this inside INFO command, here it's secondary.

Reply:
True, I forgot to add the changes to `INFO` as well. Will do!